### PR TITLE
Law: energy levels of quantum harmonic oscillator

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/quantum_mechanics/harmonic_oscillator/energy_levels.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+@validate_input(
+    quantum_number_=quantum_number,
+    angular_frequency_=angular_frequency,
+)
+@validate_output(energy_level)
+def calculate_energy_level(
+    quantum_number_: int,
+    angular_frequency_: int,
```
Comment (blackyblack) [inline symplyphysics/laws/quantum_mechanics/harmonic_oscillator/energy_levels.py:44]: `Quantity`

File: test/quantum_mechanics/harmonic_oscillator/energy_levels_test.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## The energy of the third level of a quantum harmonic oscillator for an angular frequency of
+## 3.14 rad/fs is E_3 = 7.23 eV.
+
+Args = namedtuple("Args", "n w")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    n = 3
+    w = 3.14 * units.radian / (prefixes.femto * units.second)
```
Comment (blackyblack) [inline test/quantum_mechanics/harmonic_oscillator/energy_levels_test.py:22]: Maybe use `pi` instead of 3.14?

Comment (blackyblack) [inline test/quantum_mechanics/harmonic_oscillator/energy_levels_test.py:22]: I think we were making it `Quantity` in similar test.

Comment (alesanter) [reply test/quantum_mechanics/harmonic_oscillator/energy_levels_test.py:22]: It's just a coincidence that it's `3.14`

Comment (alesanter) [reply test/quantum_mechanics/harmonic_oscillator/energy_levels_test.py:22]: Sure enough

**Review by blackyblack — APPROVED** [review]

**Pull request merged**